src/App.tsx

```diff
@@ -108,6 +114,11 @@ const App: React.FC = () => {
   return composeProviders()
     .append(Suspense, {fallback: <Loading />})
     .append(StoreProvider, {})
+    .append(PluginsContext.Provider, {
+      value: {
+        pluginItems: pluginManager.setup(),
```

Comment:
Probably should be memoized - otherwise each time there may be a new object created, re-rendering each view